internal/cobraprompt/prompt.go

```diff
@@ -9,11 +9,11 @@ import (
 	"runtime/debug"
 	"strings"
 
-	"github.com/c-bata/go-prompt"
 	"github.com/google/shlex"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/hazelcast/hazelcast-commandline-client/go-prompt"
```

Comment:
Could you move the `go-prompt` package to `internal` as well?

Reply:
Thanks, fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/58/commits/bb113e0a0f4a4c4dea35ae9618a70731aaf8d852